Cargo.toml

```diff
@@ -55,11 +55,11 @@ version = "0.3.1"
 
 [dependencies.tokio-rustls]
 optional = true
-version = "0.24.0"
+version = "0.24.1"
 
 [dependencies.webpki-roots]
 optional = true
-version = "0.23.0"
+version = "0.25.2"
```

Comment:
IIRC the webpki-roots API is exported somewhere from tokio-tungstenite, so bumping this would require releasing this as a breaking release (i.e. 0.21.0).

Reply:
I don't believe it is exported. I can only see [the one usage](https://github.com/snapview/tokio-tungstenite/blob/71d06d4495b6315181ced85b82a28044d3383854/src/tls.rs#L108) which is afaics isolated from the public api.

Reply:
I misremembered then, sorry :) Thanks for checking